Marlin/Default_Version.h

```diff
@@ -21,16 +21,48 @@
  */
 
 /**
- * This file is a placeholder for a file which could be distributed in an archive
- * It takes the place of an automatically created "_Version.h" which is generated during the build process
+ * This file is the standard Marlin version identifier file, all fields can be
+ * overriden by the ones defined on _Version.h by using the Configuration.h
+ * directive USE_AUTOMATIC_VERSIONING.
  */
 
-// #error "You must specify the following parameters related to your distribution"
-
-#if true
+/**
+ * Marlin release version identifier
+ */
 #define SHORT_BUILD_VERSION "1.1.0-RCBugFix"
-#define DETAILED_BUILD_VERSION "1.1.0-RCBugFix From Archive"
+
+/**
+ * Verbose version identifier which should contain a reference to the location
+ * from where the binary was downloaded or the source code was compiled.
+ */
+#define DETAILED_BUILD_VERSION SHORT_BUILD_VERSION " (Github)"
+
+/**
+ * The STRING_DISTRIBUTION_DATE represents when the binary file was built,
+ * here we define this default string as the date where the latest release
+ * version was tagged.
+ */
 #define STRING_DISTRIBUTION_DATE "2016-04-16 12:00"
-// It might also be appropriate to define a location where additional information can be found
-// #define SOURCE_CODE_URL  "http:// ..."
-#endif
+
+/**
+ * @todo: Missing documentation block
+ */
+#define PROTOCOL_VERSION "1.0"
+
+/**
+ * The SOURCE_CODE_URL is the location where users will find the Marlin Source
+ * Code which is installed on the device. In most cases —unless the manufacturer
+ * has a distinct Github fork— the Source Code URL should just be the main
+ * Marlin repository.
+ */
+#define DEFAULT_SOURCE_URL "https://github.com/MarlinFirmware/Marlin"
```

Comment:
Well… I really think this needs to be `SOURCE_CODE_URL`, because it's meant to override the `DEFAULT_SOURCE_URL` that's set in some pins files:
```
3DRAG
K8200 (which does it wrong)
SAV_MKI
ULTIMAIN_2
ULTIMAKER
ULTIMAKER_OLD
```